### PR TITLE
Removed opening hours and added simple-opening-hours

### PR DESCRIPTION
**Pull request by Kishore-abhimanyu**
Removed opening hours and added simple-opening-hours

### CODE REVIEW

**Comment by Kishore-abhimanyu** [comment]
can you please review and merge it?

File: package.json [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
     "jshint": "~2.11.0",
-    "opening_hours": "~3.5.0"
+    "simple-opening-hours": "^0.1.1"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/wo-ist-markt/wo-ist-markt.github.io"
   },
- "scripts": {
+  "scripts": {
```
Comment (mojoaxel) [inline package.json:17]: This new whitespace should be removed.

**Review by mojoaxel — CHANGES_REQUESTED** [review]
@Kishore-abhimanyu Thanks for trying to contribute!
- It is not enough to just replace one library with another, because they are not compatible. You also need to change the usage in the code.
- The test [fail](https://travis-ci.org/github/wo-ist-markt/wo-ist-markt.github.io/builds/737902961). Please make sure the CI turnes green after you changes.

File: package.json (hunk 1 above)
Comment (johnjohndoe) [reply package.json:17]: @Kishore-abhimanyu Do you need help with this?

Comment (johnjohndoe) [reply package.json:17]: @Kishore-abhimanyu Are you still working on this?

Comment (johnjohndoe) [reply package.json:17]: @mojoaxel Do you know the state of this pull request?

Comment (johnjohndoe) [reply package.json:17]: :bellhop_bell: @Kishore-abhimanyu, @mojoaxel Last reminder before closing this.

**Comment by johnjohndoe** [comment]
I am now closing this issue due to inactivity over the last months. If anyone is interested in the topic we can re-open it at any time. Thank you.

**Pull request closed without merging**